agent/api/ecsclient/client.go

```diff
@@ -487,27 +487,33 @@ func (client *APIECSClient) buildContainerStateChangePayload(change api.Containe
 		exitCode := int64(aws.IntValue(change.ExitCode))
 		statechange.ExitCode = aws.Int64(exitCode)
 	}
-	networkBindings := make([]*ecs.NetworkBinding, len(change.PortBindings))
-	for i, binding := range change.PortBindings {
+
+	networkBindings := []*ecs.NetworkBinding{}
+	for _, binding := range change.PortBindings {
+		if binding.BindIP == "::" && shouldExcludeIPv6PortBinding {
+			seelog.Debugf("Excluded IPv6 port binding: %v", binding)
```

Comment:
can we also log the container name/task ID if possible, so this is more useful?

Reply:
Add the container name and the task ARN to the debug log, as
```
level=debug time=xxx msg="Exclude IPv6 port binding {80 49155 :: 0} for container web in task xxxxx" module=client.go
```